### ISSUE TITLE
PINGDOM-CHECK - ID number formatting Problem

### ISSUE THREAD

**Issue by TheClonker** [monitoror/monitoror#260]
Hi, i'm playing around with Monitoror and wanted to display some Pingdom Checks.
I'm running Monitoror v3.1.0 on MacOS High Sierra 10.13.6 with Chrome 80.0.3987.132

Ping & HTTP Checks are working fine but when im trying the following example config:
`{ "type": "PINGDOM-CHECK", "label": "Pingdom Test", "params": { "id": 1000001 } },`
the Monitotor Webinterface requests the following URL: `http://127.0.0.1:8080/api/v1/pingdom/default/check?id=1.000001e%2B06`

So basically the ID being requested is formatted as 1.000001e+06 which would translate to 1000001.
The request of course fails with HTTP Error 500 `{"status":500,"message":"invalid configuration, unable to parse/check queryParams"}`. 
If i manually request the id as intended, everything works and my Pingdom result is displayed. 

So there seems to be a number formatting problem somewhere, would be cool if you could take a look :)

**Comment by TheClonker** [comment]
Thank you for the quick fix :)
